spec/unit/models/access_profile_spec.rb

```diff
@@ -11,8 +11,8 @@ module RightsAPI
     end
 
     describe "#default_order" do
-      it "returns SQL expression" do
-        expect(described_class.default_order).to be_a(Sequel::SQL::QualifiedIdentifier)
+      it "returns Array" do
```

Comment:
I'm not sure these tests bring a lot of value as-is - basically they are just testing right now that there is a default order - and I might consider just omitting them (maybe better to test that a result set ends up with the order we expect)